templates/repo/diff/box.tmpl

```diff
@@ -31,12 +31,38 @@
 			{{end}}
 			{{template "repo/diff/whitespace_dropdown" .}}
 			{{template "repo/diff/options_dropdown" .}}
+			{{if .PageIsPullFiles}}
+				<script>
+					window.config.pageData.commitInfo = {
+						issueLink: {{$.Issue.Link}},
+						queryParams: '?style={{if $.IsSplitStyle}}split{{else}}unified{{end}}&whitespace={{$.WhitespaceBehavior}}&show-outdated={{$.ShowOutdatedComments}}'
+					};
+				</script>
+				<div id="diff-commit-select">
+					{{/*
+						the following will be replaced by vue component
+						but this avoids any loading artifacts till the vue component is initialized
+					*/}}
+					<div class="ui jump dropdown basic button custom">
+						{{svg "octicon-git-commit"}}
+					</div>
+				</div>
+			{{end}}
 			{{if and .PageIsPullFiles $.SignedUserID (not .IsArchived)}}
 				{{template "repo/diff/new_review" .}}
 			{{end}}
 		</div>
 	</div>
 	{{if not .DiffNotAvailable}}
+		{{if and .IsShowingOnlySingleCommit .PageIsPullFiles}}
+			<div class="ui info message">
+				<div class="text">{{.locale.Tr "repo.pulls.showing_only_single_commit" (ShortSha .BeforeCommitID)}}</div>
```

Comment:
What's the purpose of `<div class="text">`?

Reply:
removed - thanks